server/odc-core/src/main/java/com/oceanbase/odc/core/shared/constant/ErrorCodes.java

```diff
@@ -305,6 +305,11 @@ public enum ErrorCodes implements ErrorCode {
      * version conflict
      */
     WorksheetEditVersionConflict,
+
+    /**
+     * workspace
+     */
+    WorkspaceDatabaseUserTypeMustBeAdmin,
```

Comment:
we would better add a CloudErrorCodes in cloud code repository.
CloudErrorCodes also implements ErrorCode interface